src/main/java/com/bootcamp/demo/FirebaseInitializer.java

```diff
@@ -0,0 +1,28 @@
+package com.bootcamp.demo;
+
+import com.google.firebase.FirebaseOptions;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+import static com.google.auth.oauth2.GoogleCredentials.fromStream;
+import static com.google.firebase.FirebaseApp.initializeApp;
+import static java.lang.System.getProperty;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Service
```

Comment:
Is this a service, or would it rather be a configuration class, since it only initialises the connection? I think this would be more appropriate to be done in the config class where the firestoreDb bean is also defined.